src/backy/tests/test_backy.py

```diff
@@ -31,7 +31,7 @@ def test_smoketest_internal(tmpdir):
     backup = backy.backup.Backup.init(backup_dir, 'file', source1)
 
     # Backup first state
-    backup.backup(tags={'test'})
+    backup.backup({'manual:test'}, False)
```

Comment:
Lets make False the default for the `force` parameter. In the cases where you do force it in tests, please write it as `force=True` as otherwise it's hard to understand when jumping in.